miditok/pytorch_data/datasets.py

```diff
@@ -167,8 +189,11 @@ def __init__(
         max_seq_len: int,
         tokenizer: MIDITokenizer = None,
         one_token_stream: bool = True,
+        func_to_get_labels: Callable[[Union[MidiFile, Sequence], Path], int] = None,
```

Comment:
I didn't add the item types, as it could be int (one_token_stream) but sequence of ints (multiple tracks or multi_voc) or even sequence of sequences of int (multiple tracks and multi_voc).
I think specifying all this would be quite lengthy

Reply:
Yeah alright it makes sense